docs/use-github-in-bcgov/bc-government-organizations-in-github.md

```diff
@@ -31,7 +31,6 @@ The `bcgov` organization contains public repositories that hold open-source code
 
 * Product teams working in the B.C. Gov Private Cloud PaaS OpenShift platform should use this organization unless they are given permission to work in [`bcgov-c`](#bcgov-c).
 * Any member of the `bcgov` organization can create repositories.
-* Existing members of the `bcgov` organization can invite new users to join the organization using the [Just Ask! tool](https://just-ask.developer.gov.bc.ca/). (***Users requesting membership to `bcgov` for themselves will be reviewed and may be declined***) 
```

Comment:
Should there be instructions here on how to join the bcgov org? I think we would want similar instructions as what was put into [Update README.md](https://github.com/bcgov/.github/pull/5). Maybe we want the instructions in that README to point to the instructions here instead?

Reply:
> Should there be instructions here on how to join the bcgov org? I think we would want similar instructions as what was put into [Update README.md](https://github.com/bcgov/.github/pull/5). Maybe we want the instructions in that README to point to the instructions here instead?

Hm, I like that idea. That way it's in 1 place rather than another. @jenreiher-bcgov are you alight with that?

Reply:
yes! one place is always better :)

Reply:
althhhouuughhh... do we want the one place to be here, or in the readme? 
let's be thoughtful about this, because having the instructions be in DevHub may help us educate folks on other things they may or may not know about GitHub, vs if they never click through it kind of leaves them learning about it in isolation of GitHub.

Reply:
I vote for the one place to be in the bcdg.